### ISSUE TITLE
Create my listed item page for all listed items of sellers

### ISSUE THREAD

**Issue by JiajunGao88** [cse442-at-ub/project_s23-one-belt-one-road#105]
**Task Tests**

*Test 1:*

1) Go to https://www-student.cse.buffalo.edu/CSE442-542/2023-Spring/cse-442j/login.php
2) Enter the username "asddsa" and the password "asddsaasd", then click on "Log In".
3) Verify if it is directed to https://www-student.cse.buffalo.edu/CSE442-542/2023-Spring/cse-442j/account.php
4) Click on "Listed Items" button in the center of the page.
5) Verify if it is directed to https://www-student.cse.buffalo.edu/CSE442-542/2023-Spring/cse-442j/my-listed-items.php
6) Verify if you can see existing all listed items related to account, regardless the amount of stock.
7) Go to https://github.com/cse442-at-ub/project_s23-one-belt-one-road/blob/main/my-listed-items.php
8) Verify if the line require_once 'apiFunctions.php'; is in the code for using API.

**Comment by anht-11** [comment]
Double check the part "blocked by". Should the task be blocked by another task?
